### DISCUSSION TITLE
Zoom error when switching back from portrait app.

### DISCUSSION THREAD

**Discussion by Jan-Arbus** [ppy/osu#18135] [Q&A]
![Screenshot_20220507_023244_sh ppy osulazer](https://user-images.githubusercontent.com/74862110/167253182-7d86ffcb-426b-40c2-90f7-5a2e9a15a53e.jpg)

Zoom error when switching back from portrait app.
This error may take several attempts to reproduce.

**Comment by peppy** [comment]
android? what model?

**Reply by Jan-Arbus** [reply]
> android? what model?

huawei p40
